### PR TITLE
Travis CI for all architectures

### PR DESCRIPTION
**Pull request by Wetitpig**
This script can trigger builds for all architectures in parallel.

I am not very sure if this works as I am banned from Travis CI. Thanks.

### CODE REVIEW

**Comment by xeffyr** [comment]
Should work. But I'm in doubt that builds will have same speed as before, likely it will be 1/4 per each architecture.

File: .travis.yml [hunk 1]
```diff
@@ -3,3 +3,8 @@ sudo: required
 cache: bundler
 services: docker
 before_script: mkdir debs && chmod 777 debs
+env:
+    - TRAVIS_ARCH=aarch64
+    - TRAVIS_ARCH=arm
+    - TRAVIS_ARCH=i686
+    - TRAVIS_ARCH=x86_64
```
Comment (xeffyr) [inline .travis.yml]: Hmm, environment variable will be just overwritten by last value.

Comment (xeffyr) [reply .travis.yml]: Seems working !

**Comment by xeffyr** [comment]
> I am not very sure if this works

It works: https://travis-ci.org/xeffyr/termux-packages/builds/483326809.

**Comment by fornwall** [comment]
Nice!

**Pull request merged**